mock/local-journal-non-test-activities.json

```diff
@@ -7,7 +7,7 @@
     {
       "slotDetail": {
         "slotId": 1000,
-        "start": "2020-12-01T09:00:00",
+        "start": "2021-01-27T09:00:00",
```

Comment:
I think the file might have to be manually untracked. A quick description is available in this article:
https://www.pluralsight.com/guides/how-to-use-gitignore-file
under the _What If I Already Have It Checked In?_ title